sdk/spring/azure-spring-data-cosmos/src/samples/java/com/azure/spring/data/cosmos/AppConfigurationCodeSnippet.java

```diff
@@ -48,6 +53,18 @@ public class AppConfigurationCodeSnippet extends AbstractCosmosConfiguration {
     @Value("${azure.cosmos.responseContinuationTokenLimitInKb}")
     private int responseContinuationTokenLimitInKb;
 
+    @Value("${azure.cosmos.diagnosticsThresholds.pointOperationLatencyThresholdInMS}")
+    private int pointOperationLatencyThresholdInMS;
+
+    @Value("${azure.cosmos.diagnosticsThresholds.nonPointOperationLatencyThresholdInMS}")
+    private int nonPointOperationLatencyThresholdInMS;
+
+    @Value("${azure.cosmos.diagnosticsThresholds.requestChargeThresholdInRU}")
+    private int requestChargeThresholdInRU;
+
+    @Value("${azure.cosmos.diagnosticsThresholds.payloadSizeInBytesThreshold}")
```

Comment:
NIT For consistency sake maybe azure.cosmos.diagnosticsThresholds.payloadSizeThresholdInBytes

Reply:
Done.